addons/stock_delivery/models/stock_quant_package.py

```diff
@@ -21,12 +21,9 @@ def _compute_weight_uom_name(self):
 
     def _compute_weight_is_kg(self):
         self.weight_is_kg = False
-        uom_id = self.env['product.template']._get_weight_uom_id_from_ir_config_parameter()
-        if uom_id == self.env.ref('uom.product_uom_kgm'):
-            self.weight_is_kg = True
-        self.weight_uom_rounding = uom_id.rounding
+        uom_name = self.env['product.template']._get_weight_uom_name_from_ir_config_parameter()
+        self.weight_is_kg = uom_name == 'kg'
```

Comment:
Seems kinda dangerous to me to rely on string comparisons, rather than record ids :confused: